src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,119 @@
+package mate.academy.dao;
+
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.exceptions.DataProcessingException;
+import mate.academy.model.Book;
+
+public class BookDaoImpl implements BookDao {
+    private static final String URL = "jdbc:mysql://localhost:3306/homework";
+    private static final String USER = "root";
+    private static final String PASSWORD = "vlad77kk";
```

Comment:
make a kind of Connection Util class, that will instantiate the connection between app and database, and produce the `Connection`'s by method call.